hosting/src/components/AdminComponent.jsx

```diff
@@ -60,6 +73,25 @@ const AdminComponent = () => {
     setIsLoading(false);
   };
 
+  const checkIsValid = () => {
+    setIsValid(name.length !== 0 && checkGoals() && checkTeamsIsUnique);
```

Comment:
`checkTeamsIsUnique` should be `checkTeamsIsUnique()` - this won't work if `checkTeamsIsUnique` wasn't the final check and got moved earlier :)